apps/cache-manager/src/app.ts

```diff
@@ -17,10 +17,18 @@ app.post('/:port/prerender', async (req, res) => {
 
     const remoteName = req.get('remote-name') || ''
 
+    const languageHeader = req.headers['content-language'] || 'en-GB'
+    const headers = {
+      'Content-Language': languageHeader,
```

Comment:
Is it case sensitive, just weirds me out that the line above has lower case for content-language and here it's upper

Reply:
don't think it is. made to lower case as well